.storybook/main.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  stories: [
+    '../packages/**/*.@(story|stories).@(ts|tsx)'
+  ],
+  addons: [
+    '@storybook/addon-actions/register',
+    '@storybook/addon-knobs/register'
```

Comment:
Two things here:

- I think you missed the viewport add-on. It was previously defined in the [addons.js file](https://github.com/brandingbrand/flagship/blob/develop/.storybook/addons.js#L3) and is missing from this array.
- Can you change the order of these so that the knobs add-on is first? The load order seems to impact the tab order, so the actions tab is initially shown now. Having the knobs tab initially shown seems more useful. (This may be configurable through some other option too.)